### DISCUSSION TITLE
PrismaClient is unable to be run in the browser

### DISCUSSION THREAD

**Discussion by Herman-Riah19** [prisma/prisma#17539] [Q&A]
### Bug description

In this code after to use the cli: `npx prisma generate` and I import prisma in my codebase as here:
`import { PrismaClient } from '@prisma/client'

const prisma = new PrismaClient()`

my browers run this error : Unhandled Runtime Error
Error: PrismaClient is unable to be run in the browser.
In case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by nurul3101** [comment]
Hello @Herman-Riah19 👋 

Are you using Prisma with Next.js? If yes, then can you make sure that you are using Prisma in API routes or in getServerSideProps or getStaticProps?

For reference, here's a [guide](https://www.prisma.io/nextjs) on using Next.js with Prisma.

**Comment by hozaifa4you** [comment]
I am facing the same problem with nextjs-13.3.4.

How to solve the problem

Code Here -> https://github.com/yousuf4you/similarity-clone.git

**Reply by nurul3101** [reply]
Can you please open a [new GitHub Discussion](https://github.com/prisma/prisma/discussions/new/choose) so we can debug this?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
